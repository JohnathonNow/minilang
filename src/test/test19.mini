print('', 'Hello', ' world!\n')
print('Hello
\tworld!')
let X := 10
print('X = {X}, Y = {27, 3}, Z = {2 + 2}\nL = {list(1 .. 10), ':'}')
print('\n')