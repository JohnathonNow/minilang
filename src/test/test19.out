Hello world!
Hello
	world!X = 10, Y = 1000, Z = 4
L = 1:2:3:4:5:6:7:8:9:10
